soh/src/code/z_play.c

```diff
@@ -692,6 +692,23 @@ void Gameplay_Update(GlobalContext* globalCtx) {
                                 gTrnsnUnkState = 0;
                                 R_UPDATE_RATE = 3;
                             }
+
+                            // Same code lives in z_kaleido_scope_PAL.c to save the game manually
+                            if (CVar_GetS32("gAutosave", 0)) {
+                                Gameplay_SaveSceneFlags(globalCtx);
```

Comment:
Feels like this code should be a function rather than copy and pasted our of kaleidoscope.

Reply:
Yup, definitely going to do that since any other places that we'd want to autosave would just make this problem worse.